lib/logstash/outputs/scalyr.rb

```diff
@@ -392,31 +456,37 @@ def build_multi_event_request_array(logstash_events)
 
       # generate new request if json size of events in the array exceed maximum request buffer size
       append_event = true
-      if total_bytes + event_json.bytesize > @max_request_buffer
+      add_bytes =  event_json.bytesize
+      if log_json
+        test_bytes = add_bytes + log_json.bytesize
```

Comment:
`test_bytes` is not referenced any where else.  Did you mean `add_bytes = add_bytes + log_json.bytesize`

Reply:
Yes I did, and this is why I am a fan of having to declare variables.